tests/framework/framework.go

```diff
@@ -26,6 +26,7 @@ import (
 const (
 	NsCreateTime = 30 * time.Second
 	NsDeleteTime = 5 * time.Minute
+	DefaultTimeout = 30 * time.Second
```

Comment:
The 2 above consts are specific to a resource (ns). I see `DefaultTimeout` is being used for pvcs so I think it's better to chose a name indicating that.